src/FileUtils/AgGridUtils.js

```diff
@@ -22,6 +22,21 @@ const getFlattenColumnsWithoutGroups = (columns) => {
     .filter((column) => column != null);
 };
 
+const getColumnsWithHeaderName = (columns) => {
+  if (!Array.isArray(columns)) {
+    console.warn('Columns list must be an array');
+    return [];
+  } else {
```

Comment:
Minor change: we could omit the `else` here to save one indentation level in the next part of the function :slightly_smiling_face:

Reply:
Primary school stuff :woman_facepalming: thanks ! :pray: